libraries/octree/src/Octree.h

```diff
@@ -216,13 +216,13 @@ class Octree : public QObject, public std::enable_shared_from_this<Octree>, publ
 
     // Octree importers
     bool readFromFile(const char* filename);
-    bool readFromURL(const QString& url, const bool isObservable = true, const qint64 callerId = -1); // will support file urls as well...
+    bool readFromURL(const QString& url, const bool isObservable = true, const qint64 callerId = -1, const bool isImport = false); // will support file urls as well...
     bool readFromByteArray(const QString& url, const QByteArray& byteArray);
-    bool readFromStream(uint64_t streamLength, QDataStream& inputStream, const QString& marketplaceID="");
+    bool readFromStream(uint64_t streamLength, QDataStream& inputStream, const QString& marketplaceID="", const bool isImport = false);
     bool readSVOFromStream(uint64_t streamLength, QDataStream& inputStream);
```

Comment:
```suggestion
```
Remove this line because there is no code implementing it and it looks weird that the function signature not include `cosnt bool isImport = false` like the other nearby functions.

Reply:
Oh, I looked into it and yes, indeed that does not exist anymore.